<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>composerBase</class>
 <widget class="QMainWindow" name="composerBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionSave_Project"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToggleRulers"/>
    <addaction name="actionShowFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="m_project_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QToolBar" name="m_viewTool_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPanComposer"/>
   <addaction name="actionZoomAll"/>
   <addaction name="actionZoomActual"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <widget class="QToolBar" name="m_print_toolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrint"/>
   <addaction name="actionExportAsImage"/>
   <addaction name="actionExportAsSvg"/>
   <addaction name="actionExportAsPDF"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowGrid"/>
  </widget>
  <widget class="QToolBar" name="m_ItemTools_toolBar">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectTool"/>
   <addaction name="actionAddArrow"/>
   <addaction name="actionAddShape"/>
   <addaction name="actionAddMap"/>
   <addaction name="actionAddHtml"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionNewComposer.svg</normaloff>:/images/themes/default/mActionNewComposer.svg</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionFileSave.svg</normaloff>:/images/themes/default/mActionFileSave.svg</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomIn.svg</normaloff>:/images/themes/default/mActionZoomIn.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomAll">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomFullExtent.svg</normaloff>:/images/themes/default/mActionZoomFullExtent.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom All</string>
   </property>
   <property name="toolTip">
    <string>Zoom Full Extent</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomOut.svg</normaloff>:/images/themes/default/mActionZoomOut.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomActual">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionZoomActual.svg</normaloff>:/images/themes/default/mActionZoomActual.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Actual</string>
   </property>
   <property name="toolTip">
    <string>Zoom Actual</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionFilePrint.png</normaloff>:/images/themes/default/mActionFilePrint.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print Map</string>
   </property>
  </action>
  <action name="actionExportAsImage">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveMapAsImage.png</normaloff>:/images/themes/default/mActionSaveMapAsImage.png</iconset>
   </property>
   <property name="text">
    <string>Export As Image</string>
   </property>
   <property name="toolTip">
    <string>Export As Image</string>
   </property>
  </action>
  <action name="actionExportAsSvg">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAsSVG.png</normaloff>:/images/themes/default/mActionSaveAsSVG.png</iconset>
   </property>
   <property name="text">
    <string>Export As Svg</string>
   </property>
   <property name="toolTip">
    <string>Export As Svg</string>
   </property>
  </action>
  <action name="actionToggleRulers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Rulers</string>
   </property>
   <property name="toolTip">
    <string>Show rulers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionUndo.png</normaloff>:/images/themes/default/mActionUndo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Revert last change</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionRedo.png</normaloff>:/images/themes/default/mActionRedo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Rstore last change</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/geographic.png</normaloff>:/images/themes/default/geographic.png</iconset>
   </property>
   <property name="text">
    <string>Show Grid</string>
   </property>
   <property name="toolTip">
    <string>Show grid</string>
   </property>
  </action>
  <action name="actionShowFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAddArrow">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/north_arrows/gpsarrow.svg</normaloff>:/images/north_arrows/gpsarrow.svg</iconset>
   </property>
   <property name="text">
    <string>Add Arrow</string>
   </property>
   <property name="toolTip">
    <string>Add arrow</string>
   </property>
  </action>
  <action name="actionPanComposer">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionPan.svg</normaloff>:/images/themes/default/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan Composer</string>
   </property>
   <property name="toolTip">
    <string>Pan composer</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionSelectTool">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSelectPolygon.svg</normaloff>:/images/themes/default/mActionSelectPolygon.svg</iconset>
   </property>
   <property name="text">
    <string>Select Tool</string>
   </property>
   <property name="toolTip">
    <string>Select tool</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionAddShape">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddBasicShape.png</normaloff>:/images/themes/default/mActionAddBasicShape.png</iconset>
   </property>
   <property name="text">
    <string>Add Shape</string>
   </property>
   <property name="toolTip">
    <string>Add Shape</string>
   </property>
  </action>
  <action name="actionAddMap">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddMap.png</normaloff>:/images/themes/default/mActionAddMap.png</iconset>
   </property>
   <property name="text">
    <string>Add Map</string>
   </property>
   <property name="toolTip">
    <string>Add map</string>
   </property>
  </action>
  <action name="actionAddHtml">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddHtml.png</normaloff>:/images/themes/default/mActionAddHtml.png</iconset>
   </property>
   <property name="text">
    <string>Add Html</string>
   </property>
   <property name="toolTip">
    <string>Add html</string>
   </property>
  </action>
  <action name="actionExportAsPDF">
   <property name="icon">
    <iconset resource="Resources/images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAsPDF.png</normaloff>:/images/themes/default/mActionSaveAsPDF.png</iconset>
   </property>
   <property name="text">
    <string>Export as PDF</string>
   </property>
   <property name="toolTip">
    <string>Export as PDF</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
